styles/scss/mixins/_underline-text.scss

```diff
@@ -0,0 +1,109 @@
+///  This mixin has 2 possible way of using it.
+///  First is to include in the class you want to hover over while text
+///  you want to underline needs to have attached class .underline-text
+///  or you can send your custom class name via parameter $element.
+///  Also you have to set $wrapper parameter to true.
+///  Second is to just include it in whichever element you want. In this
+///  case, parameter $wrapper should stay unchanged.
+///  Other parameters are optional and they allow you customize your transition.
+///
+/// @access public
+/// @author Karlo Volf
+/// @param {percentage} $thickness [10%] - Thickness of the line in percentage.
```

Comment:
Percentage of what?
Maybe explain how the thickness of the line will be calculated.